MAINTAINERS.md

```diff
@@ -135,16 +120,15 @@ During your review, consider the following points:
 
 ## Workflow for merging Pull Requests
 
-Check which branch the PR was made against — patches made against the `develop` branch should only
-be merged to `develop`. Patches made against the `master` branch may be merged to only `develop`, or
-both `develop` AND `master`. The criteria to use in this latter situation is: does the patch
-introduce anything new to the API? does it potentially mark a change in behavior of any sort? If the
-answer to either of these is "yes", merge only to the `develop` branch.
+Pull Requests should only be merged into the branches they were made against. When a PR contains a 
+feature and was made against an already released branch, you should work with the contributor to target
+it against the next release branch. The PR should have a Milestone matching the next release of the
+target branch (for example milestone `1.0.1` for branch `1.0.x`).
 
 To sum up:
 
-- If the commit is a bugfix, merge to **both** `master` **and** `develop`.
-- If the commit is a new feature, merge to `develop` **only**.
+- If the PR is a bugfix, it should target the lowest affected release branch
```

Comment:
I think we only want to target the lowest affected release branch for _security fixes_. For bugfixes, the latest release branch with a release created from it would be enough.

So, as an example, let's say we had a repo with the following release branches:

- 2.7.x
- 2.8.x
- 2.9.x

and tags:

- 2.7.0
- 2.7.1
- 2.7.2
- 2.8.0
- 2.8.1

We'd apply a bugfix against the 2.8.x branch in that case, and then end up issuing a 2.8.2 release.